X-AIP.md

```diff
@@ -29,17 +25,13 @@ The motivation is critical for AIPs that want to change Aave. It should clearly
 
 The technical specification should describe the syntax and semantics of any new feature.
 
-## Rationale
```

Comment:
didn't find a proposal using this. I expect ppl to just reword what they have written in `specification`